mmseg/models/necks/multilevel_neck.py

```diff
@@ -52,6 +53,12 @@ def __init__(self,
                     norm_cfg=norm_cfg,
                     act_cfg=act_cfg))
 
+    # default init_weights for conv(msra) and norm in ConvModule
+    def init_weights(self):
+        for m in self.modules():
+            if isinstance(m, nn.Conv2d):
+                xavier_init(m, distribution='uniform')
+
```

Comment:
Why use `xavier_init` for `Conv2d`?
`kaiming_init` for `ConvModule` is used in [MMCV](https://github.com/open-mmlab/mmcv/blob/master/mmcv/cnn/bricks/conv_module.py#L189).

Reply:
Followed FPN